angular.json

```diff
@@ -27,6 +27,11 @@
               "src/assets",
               "src/manifest.webmanifest"
             ],
+            "stylePreprocessorOptions": {
+              "includePaths": [
+                    "node_modules"
+               ]
+            },
```

Comment:
This is not needed. It seems to work fine without it and we should probably avoid including everything in `node_modules`.